src/appengine/private/components/upload-testcase/upload-form.html

```diff
@@ -182,6 +182,7 @@
       <div class="inline wide">
         <paper-input value="{{uploadParams.args}}" label="Command-line arguments (optional)" title="Additional command line flags to append when running this test case. Only necessary if a flag that isn't part of the job definition is required."></paper-input>
         <paper-input value="{{uploadParams.gestures}}" label="Gestures (optional)" title="A string representing gestures to use with this test case. It's generally not possible to manually create a gesture string. This tends to be used when re-uploading existing test cases which require gestures."></paper-input>
+        <paper-input value="{{uploadParams.issue_labels}}" label="Issue labels / hotlists (optional)" title="A set of label(s) / hotlist(s) to add to the issue. Should be a string of values separated by commas, e.g. value1,value2"></paper-input>
```

Comment:
maybe: `e.g. Label-One,Label-Two` ?

Reply:
It can be hotlist and hotlist can even be an integer id, so kept it generic to value1,value2